test/specs/scenarios/crossDevice.js

```diff
@@ -213,6 +216,28 @@ export const crossDeviceScenarios = async (lang) => {
         crossDeviceSubmit.clickOnSubmitVerificationButton()
         verificationComplete.verifyUIElements(copy)
       })
+
+      it('should check Submit Verification button can only be clicked once when there is no Complete step', async () => {
+        driver.get(localhostUrl + `?language=${lang}&noCompleteStep=true`)
+        welcome.primaryBtn().click()
+        documentSelector.passportIcon.click()
+        runThroughCrossDeviceFlow()
+        uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+        camera.takeSelfie()
+        confirm.confirmBtn().click()
+        crossDeviceClientSuccess.verifyUIElements(copy)
+        switchBrowserTab(0)
+        crossDeviceSubmit.documentUploadedMessage().isDisplayed()
+        crossDeviceSubmit.clickOnSubmitVerificationButton()
+        // Attempt to click button again, which is expected to fail, so catch the resulting error
+        try {
+          crossDeviceSubmit.clickOnSubmitVerificationButton()
+        } catch (e) {
```

Comment:
From my little JS understanding, if you take this catch, you still pass the test, right?
So in fact this test never seems to fail?

Reply:
Following our Slack conversation, have added the check to fail test on a successful button click.